packages/itinerary-body/src/stories/OtpRrItineraryBody.story.tsx

```diff
@@ -52,6 +54,17 @@ interface StoryWrapperProps {
   alwaysCollapseAlerts: boolean;
 }
 
+const RouteDescriptionFooterWithWaitTimes = ({ leg }: { leg: Leg }) => {
```

Comment:
Add a comment that these wait times are fake and derived from the leg travel time.